src/components/ActionMenu/components/Actions/Actions.tsx

```diff
@@ -92,26 +93,37 @@ export function Actions({actions = [], groups = []}: Props) {
         currentAvailableWidth;
 
       if (canFitAction) {
-        currentAvailableWidth -= actionWidthsRef.current[index];
+        currentAvailableWidth -=
+          actionWidthsRef.current[index] + ACTION_SPACING * 2;
```

Comment:
We weren't deducting spacing for these actions previously. Here's I'm deducting spacing * 2 because of the negative margin that button group is adding